library/src/main/java/com/heinrichreimersoftware/materialintro/app/IntroActivity.java

```diff
@@ -356,36 +356,36 @@ else if (position < currentPosition && Math.ceil(position) != currentPosition &&
         animator.start();
     }
 
-    private long calculateScrollDuration(int distance) {
+   private long calculateScrollDuration(int distance) {
         return Math.round(pageScrollDuration * (distance + Math.sqrt(distance)) / 2);
     }
     
    public void goToFirstSlide() {
-        int currentItem = pager.getCurrentItem();
+        int nearestPositionToFirstSlide = pager.getCurrentItem();
         boolean canGo = false;
```

Comment:
`canGo` is still logically wrong. Check [my previous comment](https://github.com/HeinrichReimer/material-intro/pull/155#r96389088)!